src/pluginutils.c

```diff
@@ -52,6 +52,16 @@ typedef struct
 PluginDocDataProxy;
 
 
+/** Returns the runtime API version Geany was compiled with.
+ *
+ * @return Geany's API version
```

Comment:
You could also take the docs from [my original PR](https://github.com/kugel-/geany/pull/3) ([this commit](https://github.com/kugel-/geany/pull/3/commits/d829a2f42889b1cb059cb0db02d80a675b8a2a50)).